<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Select</class>
 <widget class="QFrame" name="Select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QFrame" name="frameInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>675</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 66, 66);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelAmount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>照片总数：10301040</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNowId">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>当前照片编号：502000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNowNameTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>当前照片文件名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNowName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>IMG_0817.CR2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSource">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>照片源：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSource">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>162</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 179, 219);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>点击查看</string>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelKeyEnterTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>回车：已启用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonKeyEnter">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>220</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 179, 219);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>编辑快捷键</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKey1Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>数字键1：已启用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonKey1">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>260</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 179, 219);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>编辑快捷键</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKey2Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>数字键2：已启用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonKey2">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>300</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 179, 219);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>编辑快捷键</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKey4Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>数字键4：已启用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonKey4">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>380</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 179, 219);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>编辑快捷键</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKey3Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>数字键3：已启用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonKey3">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>340</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 179, 219);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>编辑快捷键</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHelp">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>593</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resource/Help.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>593</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resource/Exit.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>~~~</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>日志：</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameImage">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>950</width>
     <height>675</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>21</y>
      <width>950</width>
      <height>633</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(120, 120, 120);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
